packages/taucmdr/cli/commands/trial/tests/test_create.py

```diff
@@ -70,7 +70,7 @@ def test_help_arg(self):
 
     def test_no_time_metric(self):
         self.reset_project_storage()
-        argv = ['meas_no_time', '--metrics', 'PAPI_FP_INS', '--source-inst', 'never']
+        argv = ['meas_no_time', '--metrics', 'PAPI_L2_DCM', '--source-inst', 'never']
```

Comment:
Why was this PAPI metric changed? Only certain metrics are available on certain CPUs---we want to make sure we pick a widely available/common one for this. If PAPI_FP_INS was missing on the machine you were testing on let me know and I can dig into a better/more common choice